### DISCUSSION TITLE
Is there any ability today to store Ring videos locally without paying to Ring subscription ?

### DISCUSSION THREAD

**Discussion by drorya1** [dgreif/ring#1400] [Q&A]
Hello 
Using the API I see that we can interact with cloud, which means we can download the clips which are stored in cloud .
Today all my video clips are stored in cloud due to the fact I have subscription 
I am looking for cheaper alternatives, especially when Ring just raised the fee to 50 USD per year .
Any insights ?
I was thinking of Eufy doorbell .

**Comment by tsightler** [comment]
There are plenty of examples around that will start recording when a motion event is detected and save it to a file, however, circumventing subscriptions is not a primary goal of this project.  Obviously there are significant shortcomings to this approach, recording starts late (only after notification is received), etc.

I personally always encourage people that, if you don't want to pay a cloud subscription, don't purchase cloud powered products.  When a product is cloud connected and the board wants more revenue, it's just too tempting to turn these things into a subscription or raise the rate.  If a cloud powered product is somehow "free" then the future revenue is going to come from somewhere almost certainly.

**Comment by drorya1** [comment]
Thanks for the insights.
Generally speaking I totally agree and while purchasing it I said 30 USD
seems to be reasonable price to pay for storage .
Today however I am a captive customer and will probably shift from the
platform soon.
Yet wanted to know if there is a way to mimic the cloud and be the private
ring cloud of myself .
TIA for all the efforts and info

**Reply by dummbold** [reply]
drorya1,

I totally agree to you – as long as one HAS an account in the ring-cloud! I used the cloud in the 1 month free test phase and decided that i don't need it. The basic functionality without cloud is absolutely fine for me. But now my account is closed and i can't login. So your plug-in seems useless to me.

Let me explain a bit:

Actually the ring App runs on an old anonymous iPhone which is no more in use ("Sandbox"). You ask why? Because the privacy policy of the whole ring-world is so subterranean bad that it's not permitted in any way to run the App on a smartphone one uses for business here in EuropeI This is FORBIDDEN Software outside a completely private environment (i.e. family). And nobody uses his smartphone ONLY private. When the neighbor (not a family member) or the postman enters my entrance area and is caught by the camera (+ sent to the US + face recognition + date and time stamp + the product he delivers + stored + data sold and/or merged) i - as the operator of the device - have one foot in prison.

Most people in Europe don't care about this and they are not informed by ring about this issue! That's why they still sell in Europe - where it is not forbidden to sell such a product as long as it fulfills european rules of product quality. The user is responsible for the lawful use!

The ring video doorbell is (from the technical point of view) a fine product. But the context around it is inacceptable.

Bringing the video doorbell into Homebridge and HomeKit without use of the diabolical App and the invasive Cloud is definitely a violation of rings intention and maybe a violation of their terms of use - but it is compliant to the privacy laws in Europa! If you are in contact with ring you might bring this up to them. "F..ck european laws" should not be a part of the company philosophy of any company outside Europe. Some large companies have already gotten a bloody nose in this respect (Google, Apple, Alpha). And at some point the EU authorities will make an example of a smaller company ... believe me, they already have an eye on companies like ring!

BTW: The law provides for penalties of €20 million or, alternatively, 4% of annual worldwide turnover. I wouldn't call that “petty cash”!

**Reply by tsightler** [reply]
I'm not really clear what you are on about, this plugin cannot work without a Ring account as there us no method to access the camera locally, period.  It can work without a subscription and you can use tools like Scrypted to integrate with HSV.  Without a Ring account the camera is a brick, there's nothing anyone can do for that unless someone manages to hack it and replace the firmware.

**Reply by dummbold** [reply]
So my ring-Account is technically still working, but I can't access it?

**Comment by Memyself1000** [comment]
Has anybody written a script to automatically trigger local video recording as soon as a motion is detected or the bell pressed? I am aware there will be some delay before the recording starts, but I am happy to settle with this shortcoming.
